DigitalLearningSolutions.Web/Controllers/MyAccountController.cs

```diff
@@ -153,6 +158,7 @@ DlsSubApplication dlsSubApplication
                 return ReturnToEditDetailsViewWithErrors(formData, dlsSubApplication);
             }

```

Comment:
Bad place for this comment, but where we call `ProfessionalRegistrationNumberHelper.ValidateProfessionalRegistrationNumber` we are still worrying about whether a user has a delegate ID. So the validation on the PRN is still being skipped in admin-only cases.